projects/angular/src/button/button-loading/loading-button.ts

```diff
@@ -98,11 +93,4 @@ export class ClrLoadingButton implements LoadingListener {
     }
     this.clrLoadingChange.emit(state);
   }
-
```

Comment:
We want this behavior in there when a button switches from non-loading to loading so that the row of buttons does not shift when the loading spinner is narrower than the text of the button.

It seems the real issue here is that...

a) the minimum width for a button with a spinner may be too narrow to contain the spinner
b) a button that is initially rendered in the loading state isn't being set to the correct width

It may be a better strategy to cache a button's width when it is not in a loading state and update that cached width when/if the contents of the button change.